### PR TITLE
Adding new testcase in SampleTestNew

### PR DESCRIPTION
**Pull request by sumanmishra9192**

### CODE REVIEW

File: src/test/java/SampleTest.java [hunk 1]
```diff
@@ -15,5 +15,7 @@ public void testPatchRequest() {
         Response response = given().baseUri(BASE_URI).body(employee).log().all().patch().then().extract().response();
 
         System.out.println("status code: "+response.statusCode());
+        System.out.println("Test Finished");
+
```
Comment (RahulVarma23) [inline src/test/java/SampleTest.java]: remove this extra line

File: src/test/java/SampleTestNew.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+public class SampleTestNew {
+    String BASE_URI = "https://reqres.in/api/users";
+
+    @Test
+    public void testPatchRequest() {
+        Employee employee = new Employee("laxman", "qa" , 20);
+
+        Response response = given().baseUri(BASE_URI).body(employee).log().all().patch().then().extract().response();
+
+        System.out.println("status code: "+response.statusCode());
```
Comment (RahulVarma23) [inline src/test/java/SampleTestNew.java]: remove this print statement and add assertion

**Review by RahulVarma23 — CHANGES_REQUESTED** [review]

File: src/test/java/SampleTest.java (hunk 1 above)
Comment (sumanmishra9192) [reply src/test/java/SampleTest.java]: Done, Thanks for reviewing

File: src/test/java/SampleTestNew.java (hunk 2 above)
Comment (sumanmishra9192) [reply src/test/java/SampleTestNew.java]: Done. Thanks

**Review by RahulVarma23 — APPROVED** [review]

**Comment by sumanmishra9192** [comment]
Merging pull request

**Pull request merged**